Makefile

```diff
@@ -122,11 +124,17 @@ build-itest-race:
 	@$(call print, "Building itest binary for ${backend} backend.")
 	CGO_ENABLED=0 $(GOTEST) -v ./itest -tags="$(DEV_TAGS) $(RPC_TAGS) integration $(backend)" -c -o itest/itest.test$(EXEC_SUFFIX)
 
-#? install: Build and install lnd and lncli binaries, place them in $GOPATH/bin
+#? install: Build and install lnd and lncli binaries, place them in $GOPATH/bin, generate and install man pages.
 install:
 	@$(call print, "Installing lnd and lncli.")
 	$(GOINSTALL) -tags="${tags}" -ldflags="$(RELEASE_LDFLAGS)" $(PKG)/cmd/lnd
 	$(GOINSTALL) -tags="${tags}" -ldflags="$(RELEASE_LDFLAGS)" $(PKG)/cmd/lncli
+	# Invoke the 'manpages' target after installing binaries.
```

Comment:
i think let's remove this comment since it gets printed out. The bash script already prints out enough info i think :)

Reply:
SGTM. Updated 👍